--- a/drivers/net/wireless/Makefile	2018-05-16 01:05:19.000000000 -0700
+++ b/drivers/net/wireless/Makefile	2018-10-14 04:46:05.600021820 -0700
@@ -60,3 +60,9 @@
 
 obj-$(CONFIG_CW1200)	+= cw1200/
 obj-$(CONFIG_RSI_91X)	+= rsi/
+ifneq ($(CONFIG_MT_AP_SUPPORT),)
+obj-$(CONFIG_MT_AP_SUPPORT)       += mt_wifi_ap/
+endif
+ifneq ($(CONFIG_MT_STA_SUPPORT),)
+obj-$(CONFIG_MT_STA_SUPPORT)      += mt_wifi_sta/
+endif

--- a/arch/mips/Kconfig	2018-10-14 05:40:18.292024081 -0700
+++ b/arch/mips/Kconfig	2018-10-14 05:46:31.832024340 -0700
@@ -2755,3 +2755,5 @@
 source "lib/Kconfig"
 
 source "arch/mips/kvm/Kconfig"
+
+source "ralink/Kconfig"
\ No newline at end of file

--- a/drivers/mtd/maps/Makefile	2018-10-15 05:09:16.576018278 -0700
+++ b/drivers/mtd/maps/Makefile	2018-10-15 05:09:29.536018287 -0700
@@ -16,7 +16,8 @@
 obj-$(CONFIG_MTD_CK804XROM)	+= ck804xrom.o
 obj-$(CONFIG_MTD_TSUNAMI)	+= tsunami_flash.o
 obj-$(CONFIG_MTD_PXA2XX)	+= pxa2xx-flash.o
-obj-$(CONFIG_MTD_PHYSMAP)	+= physmap.o
+#obj-$(CONFIG_MTD_PHYSMAP)	+= physmap.o
+obj-$(CONFIG_MTD_PHYSMAP)       += ralink-flash.o
 obj-$(CONFIG_MTD_PHYSMAP_OF)	+= physmap_of.o
 obj-$(CONFIG_MTD_PISMO)		+= pismo.o
 obj-$(CONFIG_MTD_PMC_MSP_EVM)   += pmcmsp-flash.o
